packages/travix-ui-kit/components/stepper/stepper.js

```diff
@@ -0,0 +1,132 @@
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import React, { Component } from 'react';
+
+/**
+ * General Stepper component
+ */
+class Stepper extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      currentValue: props.initValue,
+    };
+  }
+
+  componentWillReceiveProps(newProps) {
+    if (newProps.initValue !== this.state.currentValue) {
```

Comment:
This will set `this.state.currentValue` to `newProps.initValue` if they are not the same and any prop is changed. You should also check if `newProps.initValue !== this.props.initValue`.